packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -44,6 +45,19 @@ const chatMessageStyles: ComponentSlotStylesInput<ChatMessageProps, ChatMessageV
       textDecoration: 'underline',
     },
   }),
+  badge: ({ props: p, variables: v }) => {
+    const sidePosition = p.badgePosition === 'start' ? 'left' : 'right'
+    return {
+      boxShadow: v.badgeShadow,
+      position: 'absolute',
+      height: pxToRem(24),
```

Comment:
This breaks when you change icon size. Any chance this could work without absolute values?

Something like:
```jsx
      boxShadow: v.badgeShadow,
      position: 'absolute',
      padding: pxToRem(4),
      height: 'auto',
      width: 'auto',
      borderRadius: '50%',
      top: pxToRem(4),
      zIndex: '1',
      [sidePosition]: 0,
      transform: p.badgePosition === 'start' ? 'translateX(-50%)' : 'translateX(50%)',
```

Reply:
+1 for using transform

Reply:
Thanks Miro, I've changed the styles, work with every size of icon, and the default is according to the specs. I've added badge in the styles ChatItem example as well. Let me know if there is anything else.